docs/epayment.v1.yml

```
@@ -301,34 +260,23 @@ components:
           example: CUSTOMER_NOT_PRESENT
           default: CUSTOMER_NOT_PRESENT
         industryData:
-          description: Additional data related to the transaction.
+          description: Additional compliance data related to the transaction.
           type: object
           properties:
-            airline:
+            airlineData:
               $ref: '#/components/schemas/AirlineData'
-        logistics:
-          $ref: '#/components/schemas/Logistics'
-        orderInformation:
-          $ref: '#/components/schemas/OrderInformation'
+        receipt:
+          $ref: 'https://raw.githubusercontent.com/vippsas/vipps-order-management-api/2.3.0/docs/swagger.yaml#/components/schemas/Receipt'
         paymentMethod:
           $ref: '#/components/schemas/PaymentMethod'
         profile:
-          $ref: '#/components/schemas/Profile'
-        pspReference:
-          $ref: '#/components/schemas/PspReferenceType'
-        redirectUrl:
-          type: string
-          format: uri
-          example: 'https://landing.vipps.no?token=abc123'
-          description: 'The url you should redirect the user to to continue with the payment. '
-          readOnly: true
+          $ref: '#/components/schemas/ProfileRequest'
         reference:
-          $ref: '#/components/schemas/ReferenceType'
+          $ref: '#/components/schemas/Reference'
         returnUrl:
           type: string
+          pattern: '^(https:\/\/)[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$'
           example: 'https://example.io/redirect?orderId=abcc123'
```

example.com is an example domain, and our usual orderId is here: https://github.com/vippsas/vipps-rfc/blob/master/docs/0001-api-guidelines.md#examples-of-data-types-and-values
```suggestion
          example: 'https://example.com/redirect?orderId=acme-shop-123-order123abc'
```